pipeline/notifications/__init__.py

```diff
@@ -1,2 +1,4 @@
 from pipeline.notifications.check_message import *
 from pipeline.notifications.notifier import *
+from pipeline.notifications.render_check_message import *
```

Comment:
There's a slight discrepancy between your class names and your file names. Maybe rename the renderer and notifier files to match the class name?

Reply:
Yeah I see what you mean. In fact, I'll go ahead and rename all of the notifications files so that they're more consistent in their naming.